client/verta/tests/unit_tests/test_runtime.py

```diff
@@ -140,7 +140,7 @@ def test_exception_on_nesting_context_managers() -> None:
         with runtime.context():
             with runtime.context():
                 pass
-    assert err.value.args[0] == " Nesting an instance of verta.runtime.context() inside" \
+    assert err.value.args[0] == "nesting an instance of verta.runtime.context() inside" \
```

Comment:
This fixes `unit_tests/test_runtime.py::test_exception_on_nesting_context_managers` after I changed the exception message in https://github.com/VertaAI/modeldb/pull/3567 but neglected to update this test.